packages/extract-text2/index.js

```diff
@@ -31,8 +31,8 @@ function extractText (outputFilePattern, fileType) {
             test: context.fileType(fileType),
             exclude: loaderConfig.exclude,
             loader: plugin.extract({
-              fallbackLoader: 'style-loader',
-              loader: nonStyleLoaders
+              fallback: 'style-loader',
+              use: nonStyleLoaders
             }),
```

Comment:
You can see it here: https://github.com/cyclejs-community/create-cycle-app/blob/39296f083f93cafbb31afbf928c7965ed0329228/packages/cycle-scripts/configs/webpack.config.template.js#L4